2.x-activity-module/mod/aspirelists/launch.php

```diff
@@ -20,5 +20,11 @@
 
 $list->cmid = $cm->id;
 aspirelists_add_lti_properties($list);
-lti_view($list);
-
+if($CFG->version >= 2015111600) {
```

Comment:
It simply increments the last two digits in that number, as per: https://docs.moodle.org/dev/Releases

There is an alternative of using ``$CFG->branch`` as this PR is using in version.php.  I'm not sure if there is a particular advantage to eitehr approach.  I think version looks reasonable here, it's the value that the google docs are using as examples for how to populate the db upgrade file.